Formula/cassowary.rb

```diff
@@ -0,0 +1,26 @@
+class Cassowary < Formula
+  desc "Modern cross-platform HTTP load-testing tool written in Go"
+  homepage "https://github.com/rogerwelin/cassowary"
+  url "https://github.com/rogerwelin/cassowary.git",
+    :tag      => "v0.10.0",
+    :revision => "cff77858f8d2eef6cbd7d7b7ef4999b2e6c54931"
+  head "https://github.com/rogerwelin/cassowary.git"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
```

Comment:
no need to have bottle block for new formula
```suggestion
```

Reply:
@chenrui333 fixed! (squashed into one commit)